Model/Admin/Source/ExtraAttributesList.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Wizzy\Search\Model\Admin\Source;
+
+use Magento\Catalog\Model\ResourceModel\Product\Attribute\CollectionFactory;
+
+class ExtraAttributesList
+{
+    protected $collectionFactory;
+
+    public function __construct(CollectionFactory $collectionFactory)
+    {
+        $this->collectionFactory = $collectionFactory;
+    }
+
+    public function toOptionArray()
+    {
+        $attributes = $this->collectionFactory->create();
```

Comment:
Move this code into `AttributesManager`  Same as Model/Admin/Source/AttributesList.php